### PR TITLE
SOFT4 Tournament page

### PR DESCRIPTION
**Pull request by WalterToro**
Its actually happening boiz

---

### CODE REVIEW

File: wiki/Tournaments/SOFT/4/en.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
+---
+tags:
+- SOFT4
+- SOFT 4
+---
+# Springtime Osu!mania Free-for-all Tournament 4
+
+![SOFT4 Banner](img/banner.png)
```
Comment (fajar13k) [inline wiki/Tournaments/SOFT/4/en.md]: Aren't we suppose to do `Logo` naming on this?

File: wiki/Tournaments/SOFT/4/en.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+> SOFT 4 When?, now!
+
+**SOFT 4**, short for **Springtime Osu!mania Free-for-all Tournament 4**, is a one-on-one, country-independent tournament. As the title indicates, this tournament is for the osu!mania game mode. More specifically, this tournament is an osu!mania 4-key tournament.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | --- |
+| Registration Phase   | 2019-03-06/2019-03-22 |
```
Comment (fajar13k) [inline wiki/Tournaments/SOFT/4/en.md]: Remove unnecesary whitespaces (?)

File: wiki/Tournaments/SOFT/4/en.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+| Commentators | ![][flag_US] [Daikyi](https://osu.ppy.sh/users/811832), ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656), ![][flag_PL] [Kamikaze](https://osu.ppy.sh/users/2124783), ![][flag_AU] [Knif3rism](https://osu.ppy.sh/users/4145830), ![][flag_SE] [laughingcomet](https://osu.ppy.sh/users/3432877), ![][flag_AU] [nonamegohst](https://osu.ppy.sh/users/4384859), ![][flag_AU] [PingLaw](https://osu.ppy.sh/users/5401523), ![][flag_AU] [PotassiumF](https://osu.ppy.sh/users/4247722), ![][flag_ID] [RemFangirl](https://osu.ppy.sh/users/5767941), ![][flag_US] [Ruins](https://osu.ppy.sh/users/1408132), ![][flag_US] [stupud man](https://osu.ppy.sh/users/2141612), ![][flag_US] [TheToaphster](https://osu.ppy.sh/users/7616811) |
+| Graphic Designers | ![][flag_PH] [OsuMe65](https://osu.ppy.sh/users/852867), ![][flag_PL] [W D Gaster](https://osu.ppy.sh/users/1475814), ![][flag_US] [Ruins](https://osu.ppy.sh/users/1408132), ![][flag_CL] [Valelup](https://osu.ppy.sh/users/6497014) |
+| Statisticians | ![][flag_PH] [SurfChu85](https://osu.ppy.sh/users/4469895) |
```
Comment (fajar13k) [inline wiki/Tournaments/SOFT/4/en.md]: Statistician, sad SurfChu

File: wiki/Tournaments/SOFT/4/en.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+| Finals               | 2019-06-08/2019-06-09 |
+| Grand Finals         | 2019-06-15/2019-06-16 |
+
+## Prizes
+
+*Prizes for this tournament are to be decided.*
+
+## Organization
+
+| Position | Member |
```
Comment (fajar13k) [inline wiki/Tournaments/SOFT/4/en.md]:
```suggestion
| Position | Member(s) |
```

File: wiki/Tournaments/SOFT/4/en.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+| Quarterfinals        | 2019-05-25/2019-05-26 |
+| Semifinals           | 2019-06-01/2019-06-02 |
+| Finals               | 2019-06-08/2019-06-09 |
+| Grand Finals         | 2019-06-15/2019-06-16 |
+
+## Prizes
+
+*Prizes for this tournament are to be decided.*
+
+## Organization
```
Comment (fajar13k) [inline wiki/Tournaments/SOFT/4/en.md]:
```suggestion
## Organisation
```

File: wiki/Tournaments/SOFT/4/en.md [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+| Commentators | ![][flag_US] [Daikyi](https://osu.ppy.sh/users/811832), ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656), ![][flag_PL] [Kamikaze](https://osu.ppy.sh/users/2124783), ![][flag_AU] [Knif3rism](https://osu.ppy.sh/users/4145830), ![][flag_SE] [laughingcomet](https://osu.ppy.sh/users/3432877), ![][flag_AU] [nonamegohst](https://osu.ppy.sh/users/4384859), ![][flag_AU] [PingLaw](https://osu.ppy.sh/users/5401523), ![][flag_AU] [PotassiumF](https://osu.ppy.sh/users/4247722), ![][flag_ID] [RemFangirl](https://osu.ppy.sh/users/5767941), ![][flag_US] [Ruins](https://osu.ppy.sh/users/1408132), ![][flag_US] [stupud man](https://osu.ppy.sh/users/2141612), ![][flag_US] [TheToaphster](https://osu.ppy.sh/users/7616811) |
+| Graphic Designers | ![][flag_PH] [OsuMe65](https://osu.ppy.sh/users/852867), ![][flag_PL] [W D Gaster](https://osu.ppy.sh/users/1475814), ![][flag_US] [Ruins](https://osu.ppy.sh/users/1408132), ![][flag_CL] [Valelup](https://osu.ppy.sh/users/6497014) |
+| Statisticians | ![][flag_PH] [SurfChu85](https://osu.ppy.sh/users/4469895) |
+
+## Links
+
+- [Discussion Thread](https://osu.ppy.sh/community/forums/posts/877866)
```
Comment (fajar13k) [inline wiki/Tournaments/SOFT/4/en.md]:
```suggestion
- [Discussion thread](https://osu.ppy.sh/community/forums/posts/877866)
```
Just lowercase

File: wiki/Tournaments/SOFT/4/en.md [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+- [Livestream](https://www.twitch.tv/osusoft)
+- [Discord Server](https://discord.gg/qd575a3)
```
Comment (fajar13k) [inline wiki/Tournaments/SOFT/4/en.md]:
```suggestion
- [SOFT Discord server](https://discord.gg/qd575a3)
```

File: wiki/Tournaments/SOFT/4/en.md [hunk 8]
```diff
@@ -0,0 +1,245 @@
...
+- **[Registration Form](https://docs.google.com/forms/d/e/1FAIpQLScWH5sExtBaftjaoUzlghMNWbw8iOhtdvwEr7P1xmBrM5DwVQ/viewform)**
+
+---------------------------
+
+## Rules
+
+### Tournament Organization
```
Comment (fajar13k) [inline wiki/Tournaments/SOFT/4/en.md]:
```suggestion
### Tournament Organisation
```

File: wiki/Tournaments/SOFT/4/en.md [hunk 9]
```diff
@@ -0,0 +1,245 @@
...
+- SOFT4
+- SOFT 4
+---
+# Springtime Osu!mania Free-for-all Tournament 4
+
+![SOFT4 Logo](img/logo.png)
+
+> SOFT 4 When?, now!
+
+**SOFT 4**, short for **Springtime Osu!mania Free-for-all Tournament 4**, is a one-on-one, country-independent tournament. As the title indicates, this tournament is for the osu!mania game mode. More specifically, this tournament is an osu!mania 4-key tournament.
```
Comment (TPGPL) [inline wiki/Tournaments/SOFT/4/en.md]:
```suggestion
The **Springtime Osu!mania Free-for-all Tournament 4** (***SOFT 4***), is a one-on-one, country-independent osu!mania 4-key tournament.
```

File: wiki/Tournaments/SOFT/4/en.md [hunk 10]
```diff
@@ -0,0 +1,245 @@
...
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | --- |
```
Comment (TPGPL) [inline wiki/Tournaments/SOFT/4/en.md]:
```suggestion
| --: | :-- |
```

File: wiki/Tournaments/SOFT/4/en.md [hunk 11]
```diff
@@ -0,0 +1,245 @@
...
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/877866)
+- [Livestream](https://www.twitch.tv/osusoft)
+- [SOFT Discord server](https://discord.gg/qd575a3)
+- **[Registration form](https://docs.google.com/forms/d/e/1FAIpQLScWH5sExtBaftjaoUzlghMNWbw8iOhtdvwEr7P1xmBrM5DwVQ/viewform)**
+
+---------------------------
+
+## Rules
+
+### Tournament OrganiSation
```
Comment (TPGPL) [inline wiki/Tournaments/SOFT/4/en.md]:
```suggestion
### Tournament Organisation
```

**Pull request merged**